_layouts/post.html

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+{% include back-to-top.html %}
+
+<p><a href="{{site.baseurl}}/blog">Blog</a> <i class="fas fa-chevron-right"></i> {{ page.title }}</p>
+
+
+<div class="home post-page">
+  {{ page.date | date: '%B %d, %Y' }} 
+
+  <h1>{{ page.title }}</h1>
+  
+  <div class="byline-wrapper">
+    {% assign author = site.data.authors[page.author] %}
+    {% if author.emailhash %}
+    <img class="headshot" src="https://www.gravatar.com/avatar/{{ author.emailhash }}">
+    {% endif %}
+    <p class="by">By &nbsp;</p>
```

Comment:
Looks like we have double space here. Looks a little odd with that extra space:D

![Screenshot from 2022-10-19 16-08-36](https://user-images.githubusercontent.com/68053619/196793623-01e9fa66-607a-4f58-96b3-f6828313b9e9.png)